lib/samson/syslog_formatters.rb

```diff
@@ -4,11 +4,21 @@
 module Samson
   class SyslogFormatter < Syslog::Logger::Formatter
     def call(severity, timestamp, _progname, message)
+      message_h =
+        begin
+          message = message.to_json unless message.is_a?(String)
```

Comment:
so message comes in as a string or hash ? ... blow up when it's anything else ?
.... does it ever come in as string with json ? / can we avoid that ?

Reply:
Not all times, we may also get a different object as input. 
I wish not to blow up because this formatter is now used by external gems, we just try to prettify the messages as much as possible, if not then we just squish them and embed in the message attributes of hash.

Reply:
converting things to json and then parsing them right after is inefficient, how can we avoid that ?
who is passing json-ified objects into the logger / can we opt out of supporting that ?

Reply:
why do we have to squish things ? .... that has the potential to modify important message details, so I'd rather avoid that if possible

Reply:
ok, made alterations.